Build error: QDialog

Hey I am new to Raspbian + Linux in general so this could very well be user error. 
But when following the steps for setup from the wiki I get the following error on build. 

Appreciate the help! 

### Operating Systems ###
Server: Windows10:1809
Client: Linux raspberrypi 4.19.58-v7+ #1245 SMP Fri Jul 12 17:25:51 BST 2019 armv7l GNU/Linux

### Barrier Version ###
Current

### Steps to reproduce bug ###
On build from new pull from repo, fresh OS build... etc. 
Run Linux Procedure for building (from Wiki)

When running "sudo make install" the following error is produced. 
[ 78%] Building CXX object src/gui/CMakeFiles/barrier.dir/src/AboutDialog.cpp.o
In file included from /home/pi/barrier/build/barrier/src/gui/src/AboutDialog.cpp:19:
/home/pi/barrier/build/barrier/src/gui/src/AboutDialog.h:23:10: fatal error: QDialog: No such file or directory
 #include <QDialog>
          ^~~~~~~~~
compilation terminated.
make[2]: *** [src/gui/CMakeFiles/barrier.dir/build.make:63: src/gui/CMakeFiles/barrier.dir/src/AboutDialog.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:917: src/gui/CMakeFiles/barrier.dir/all] Error 2
make: *** [Makefile:152: all] Error 2

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -349,6 +349,12 @@ else()
     set (OPENSSL_LIBS ${lib_ssl} ${lib_crypto})
 endif()
 
+
+# Check we have the *required* Qt5 libs.
+find_package(Qt5Core REQUIRED)
+find_package(Qt5Network REQUIRED)
+find_package(Qt5Widgets REQUIRED)
+
 #
 # Configure_file... but for directories, recursively.
 #

